### PR TITLE
Build release-0.2 CAPI branch

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

/assign @Katharine @ncdc @detiber

### CODE REVIEW

File: config/jobs/image-pushing/k8s-staging-cluster-api.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ postsubmits:
       decorate: true
       branches:
         - ^master$
+        - ^release-0.2$
```
Comment (ncdc) [inline config/jobs/image-pushing/k8s-staging-cluster-api.yaml]: If you specifically want just this one branch:

```suggestion
        - ^release-0\.2$
```

But what if we did this instead, to cover all release branches?
```
^release-
```

Comment (detiber) [reply config/jobs/image-pushing/k8s-staging-cluster-api.yaml]: I don't think the referenced script is present in the release-0.1 branch

Comment (ncdc) [reply config/jobs/image-pushing/k8s-staging-cluster-api.yaml]: Which script?

Comment (detiber) [reply config/jobs/image-pushing/k8s-staging-cluster-api.yaml]: It's missing the cloud-build.yaml and the make targets used by it.

**Pull request merged**